rust_src/src/frames.rs

```diff
@@ -31,9 +35,18 @@ impl LispFrameRef {
         !self.terminal.is_null()
     }
 
+    pub fn focus_frame(self) -> LispObject {
+        self.focus_frame.into()
+    }
+
     // Awaiting Wilfred#1264
```

Comment:
Is this comment still valid? If not, let's drop it too.

Reply:
PR still open, I've pulled code from it to simplify merge...